docs/bash_to_xsh.rst

```diff
@@ -138,3 +138,16 @@ there is `bashisms xontrib <https://xon.sh/xontribs.html#bashisms>`_:
     >>> !!
     echo
 
+Work with bash snippets? Take a look at `xontrib-sh <https://xon.sh/xontribs.html#sh>`_:
```

Comment:
I am a little uncomfortable with the syntax here and should probably move this into the xontrib description

Reply:
text removed, examples combined